tests/test_permissions.py

```diff
@@ -716,3 +716,59 @@ def has_object_permission(self, request, view, obj):
         composed_perm = (IsAuthenticatedUserOwner | permissions.IsAdminUser)
         hasperm = composed_perm().has_object_permission(request, None, None)
         assert hasperm is False
+
+    def test_operand_holder_is_hashable(self):
+        assert hash((permissions.IsAuthenticated & permissions.IsAdminUser))
```

Comment:
May fail if `hash` returns zero. The proper check would be that no exception is thrown. (E.g., via `>= 0`)